container/go/cmd/digester_index/digester_index.go

```
@@ -0,0 +1,251 @@
+	}
+
+	if args.OutputFilePath == "" {
+		return nil, errors.New("required options -dst was not specified")
+	}
+
+	if args.Format == "" {
+		return nil, errors.New("required options -format was not specified")
+	}
+
+	for fl.NArg() > 0 {
+		var imageArgs imageArgs
+
+		argv := fl.Args()
+
+		fl = flag.NewFlagSet(appName, flag.ContinueOnError)
+		fl.StringVar(
+			&imageArgs.Platform.Os, "os", "",
+			"OS of the Docker image.",
+		)
+		fl.StringVar(
+			&imageArgs.Platform.Arch, "arch", "",
+			"Architecture of the Docker image.",
+		)
+		fl.StringVar(
+			&imageArgs.Platform.Variant, "variant", "",
+			"Architecture variant of the Docker image.",
+		)
+
+		fl.StringVar(
+			&imageArgs.Config, "config", "",
+			"Path to the image config.json. Required if --tarball was not specified.",
+		)
+		fl.Var(
+			(*utils.ArrayStringFlags)(&imageArgs.Layers), "layer",
+			"One or more layers with the following comma separated values (Compressed layer tarball, Uncompressed layer tarball, digest file, diff ID file)."+
+				"e.g., --layer layer.tar.gz,layer.tar,<file with digest>,<file with diffID>.",
+		)
+		fl.StringVar(
+			&imageArgs.Tarball, "tarball", "",
+			"Path to the image tarball as generated by docker save. Required if --config was not specified.",
+		)
+		fl.StringVar(
+			&imageArgs.Manifest, "manifest", "",
+			"Path to the manifest of the base image. This should be the very first image in the chain of images and is only really required for windows images with a base image that has foreign layers.",
+		)
+
+		if err := fl.Parse(argv); err != nil {
+			return nil, err
+		}
+
+		if imageArgs.Tarball == "" && imageArgs.Config == "" {
+			return nil, errors.New("neither --tarball nor --config was specified")
+		}
+
+		args.Images = append(args.Images, imageArgs)
+	}
+
+	return &args, nil
+}
+
+func main() {
+	args, err := parseArgs(os.Args)
+	if err != nil {
+		if err == flag.ErrHelp {
+			os.Exit(0)
+		}
+
+		_, _ = fmt.Fprintf(os.Stderr, "Error: %v\n", err)
```

is the `_, _ =` really needed?

---

Yes, and no :). The main goal to explicitly ignore all errors is to avoid asking the question about if the error is voluntarily ignored or if it's an omission.
And I agree with you, for all standard and popular functions the added value is very limited.